mache/spack/chrysalis_gnu_openmpi.yaml

```diff
@@ -86,65 +86,65 @@ spack:
       buildable: false
     gcc:
       externals:
-      - spec: gcc@9.2.0
-        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-9.3.0/gcc-9.2.0-ugetvbp
+      - spec: gcc@11.2.0
+        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-9.3.0/gcc-11.2.0-bgddrif
         modules:
-        - gcc/9.2.0-ugetvbp
+        - gcc/9.2.0-bgddrif
       buildable: false
     openmpi:
       externals:
-      - spec: openmpi@4.1.3
-        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-9.2.0/openmpi-4.1.3-sxfyy4k
+      - spec: openmpi@4.1.6
+        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-11.2.0/openmpi-4.1.6-ggebj5o
         modules:
-        - openmpi/4.1.3-sxfyy4k
+        - openmpi/4.1.6-ggebj5o
       buildable: false
-    intel-mkl:
+    intel-oneapi-mkl:
       externals:
-      - spec: intel-mkl@2020.4.304
-        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-9.2.0/intel-mkl-2020.4.304-n3b5fye/compilers_and_libraries_2020.4.304
+      - spec: intel-oneapi-mkl@2022.1.0
+        prefix: /gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-11.2.0/intel-oneapi-mkl-2022.1.0-w4kgsn4/compilers_and_libraries_2022.1.0
```

Comment:
It turns out this path changed to. I think the right path is just:
```
/gpfs/fs1/soft/chrysalis/spack/opt/spack/linux-centos8-x86_64/gcc-11.2.0/intel-oneapi-mkl-2022.1.0-w4kgsn4
```
I found this out by loading the old module on chrysalis:
```
module load gcc/9.2.0-ugetvbp intel-mkl/2020.4.304-n3b5fye
module show intel-mkl
```
and, in a new terminal, the new ones:
```
module load gcc/11.2.0-bgddrif intel-oneapi-mkl/2022.1.0-w4kgsn4
module show intel-oneapi-mkl
```
It's a little unclear because a lot changed but the path above is my best bet.  We'll test it out before we merge this.